documents/dispute.md

```diff
@@ -0,0 +1,299 @@
+    }
+  ],
+  "action": "draft"
+})
+```
+
+**Parameters:**
+
+| Name            | Type    | Description                                                                                                                                                                     |
+|-----------------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| disputeId*      | string  | The unique identifier of the dispute.                                                                                                                                           |
+| amount          | integer | The amount being contested. If the contest amount is not mentioned, we will assume it to be a full dispute contest.                                                             |
+| summary         | string  | The explanation provided by you for contesting the dispute. It can have a maximum length of 1000 characters.                                                                    |
+| shipping_proof  | array   | List of document ids which serves as proof that the product was shipped to the customer at their provided address. It should show their complete shipping address, if possible. |
+| others          | array   | All keys listed [here](https://razorpay.com/docs/api/disputes/contest) are supported                                                                                            |
+| action          | string  | Possible value is `draft` or `submit`                                                                                                                                           |
+
+**Response:**
+```json
+{
+  "id": "disp_AHfqOvkldwsbqt",
+  "entity": "dispute",
+  "payment_id": "pay_EsyWjHrfzb59eR",
+  "amount": 10000,
+  "currency": "INR",
+  "amount_deducted": 0,
+  "reason_code": "chargeback",
+  "respond_by": 1590604200,
+  "status": "open",
+  "phase": "chargeback",
+  "created_at": 1590059211,
+  "evidence": {
+    "amount": 5000,
+    "summary": "goods delivered",
+    "shipping_proof": [
+      "doc_EFtmUsbwpXwBH9",
+      "doc_EFtmUsbwpXwBH8"
+    ],
+    "billing_proof": null,
+    "cancellation_proof": null,
+    "customer_communication": null,
+    "proof_of_service": null,
+    "explanation_letter": null,
+    "refund_confirmation": null,
+    "access_activity_log": null,
+    "refund_cancellation_policy": null,
+    "term_and_conditions": null,
+    "others": [
+      {
+        "type": "receipt_signed_by_customer",
+        "document_ids": [
+          "doc_EFtmUsbwpXwBH1",
+          "doc_EFtmUsbwpXwBH7"
+        ]
+      }
+    ],
+    "submitted_at": null
+  }
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+### Contest a Dispute (Submit)
+
+```rb
+# Use this API sample code for submit
+
+Razorpay::Dispute.fetch(disputeId).contest({
```

Comment:
can you update this ?

Reply:
updated

https://github.com/razorpay/razorpay-ruby/blob/2e0b54f61b54b10dff29bb1d178a7fcf95b5effe/documents/dispute.md?plain=1#L234